resources/winsetup/Yarn.wxs

```diff
@@ -36,6 +36,15 @@
 					Action="set"
 					System="yes"
 				/>
+				<Environment
+					Id="BinPath"
+					Name="PATH"
+					Value="[LocalAppDataFolder]Yarn\.bin"
+					Permanent="no"
+					Part="last"
+					Action="set"
+					System="yes"
```

Comment:
Can you, maybe, set the path to be literally `%LocalAppData%/Yarn/.bin`, and hope the shell expands it when evaluating the `PATH`?

This, of course, will definitely not work on windows xp.